operators/pkg/controller/elasticsearch/initcontainer/prepare_fs_script_test.go

```diff
@@ -29,11 +28,8 @@ func TestRenderScriptTemplate(t *testing.T) {
 							Target: "/usr/share/elasticsearch/users"}}},
 			},
 			wantSubstr: []string{
-				"$PLUGIN_BIN install --batch repository-s3",
-				"$PLUGIN_BIN install --batch repository-gcs",
 				"mv /usr/share/elasticsearch/config/* /mnt/elastic-internal/elasticsearch-config-local/",
 				"mv /usr/share/elasticsearch/bin/* /mnt/elastic-internal/elasticsearch-bin-local/",
```

Comment:
I think we don't need these 2 anymore (see above)

Reply:
Actually I think I need to revert the last commit and restore the plugin mount again. Otherwise it will be really tricky for users to make this work correctly

Reply:
Hm you're right.